src/public/publicAPIs.ts

```diff
@@ -87,8 +86,30 @@ export function initialize(hostWindow: any = window): AppInitializationEvent {
       GlobalVars.isFramelessWindow = false;
     };
   }
+}
+
+/**
```

Comment:
NIT: Consider putting all of the notifyApp*** APIs in their own namespaces (e.g. like settings, tasks, etc.)